test/integration/windows/windows_suite_test.go

```diff
@@ -72,6 +79,9 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+	By("deleting the configmap")
+	configMapWrapper.DeleteConfigMap(frameWork.K8sClient, ctx, configMap)
```

Comment:
can we revert back to the older state if ConfigMap already existed (as in future it might contain CNI related variables)

Reply:
Resolved in the last commit.